pennylane/vqe/vqe.py

```diff
@@ -45,6 +46,21 @@ class Hamiltonian:
 
     .. seealso:: :class:`~.ExpvalCost`, :func:`~.molecular_hamiltonian`
 
+    .. Warning::
+
+        Hamiltonians can be constructed using Pythonic arithmetic operations. For example:
+
+        >>> qml.PauliX(0) + 2 * qml.PauliZ(0) @ qml.PauliZ(1)
```

Comment:
I wonder if the `invisible` transform will solve this issue.

Reply:
Maybe, I'll try adding it!